### ISSUE TITLE
Basic Item Filter recipe change

### ISSUE THREAD

**Issue by Methes** [GTNewHorizons/NewHorizons#1203]
With the new update the process of creating a Basic Item Filter has changed drastically.

Before you could use the Platinum -> Fl. Extractor -> Autoclave + Carbon dust -> Assembling machine x2 to create something quite essential for more complicated networks.

Now you can't use molten Platinum but instead you need Epoxid, which is also ridiculous to get.

I often feel like someone decides a certain recipe is too easy so he makes it harder without realizing other things might depend on the whole chain and break it.

**Comment by Dream-Master** [comment]
I change the recipe because the recipe before was wrong and I like more realistic recipes. I have a discussion with The GT++ dev and we come together to the point that Platinum, Iridium not make sense. My recipe give a 100% chance to get an item and not lose it.
So is the amount of Epoxide it use to high? Or did need Epoxide an additional recipes.
I won’t add this unrealistic Platinum recipe back it not make sense.

@Methes if you have a better realistic suggestion of a new or additional recipe post it here right under my comment.

**Comment by Bryfer** [comment]
I would suggest using Molten Polyethylene instead, or as an alternative more costly recipe.
Molten Polyethylene is used for alot of MV-HV items while Epoxide, or Epichlorohydrin which is used to created Epoxide. (Epichlorohydrin) is used to create EV+ voltage tier machines starting with the Empty Processor board which unlocks the creation of Data Control Circuits (EV), and Energy Flow Circuit (IV)

By Limiting Carbon fibre to only be crafted using Molten Epoxid, you force players to start producing EV+ liquids in MV-HV tier and therefore pushes Ender IO and Nano armor into EV tier instead of MV-HV
(Yes I know that you dont have to be in EV, but the machines you create only to make carbon fibre will only stand and rust, unless you need more carbon fibre or reach ev+)

Epoxide and/or Epichlorohydrin can easily be created in the beginning of EV tier, once you unlock distillation tower.

In other words I'd like;
Molten Polyehtylene to be used up to HV tier.
Polytetrafluoroethylene (Epichlorohydrin) to be used EV+.

**Comment by Dream-Master** [comment]
Ok will add recipes.

**Comment by Methes** [comment]
Yes, that sounds reasonable. *thumbs up*

**Comment by Dream-Master** [comment]
![image](https://cloud.githubusercontent.com/assets/2970688/23873966/441b48ba-0834-11e7-9a15-dfea435755dd.png)
![image](https://cloud.githubusercontent.com/assets/2970688/23873972/4a32528e-0834-11e7-82d5-1d607e3fb471.png)
![image](https://cloud.githubusercontent.com/assets/2970688/23873974/4e2700d8-0834-11e7-9e2c-d548a465e268.png)

alternative item filter recipe with 64 fine steel wire
![image](https://cloud.githubusercontent.com/assets/2970688/23873998/60607f72-0834-11e7-83a3-27034d49600a.png)
